pysatModels/tests/test_utils_extract.py

```diff
@@ -789,3 +862,73 @@ def teardown(self):
         del self.inst, self.model, self.input_args, self.out, self.in_kwargs
 
         return
+
+
+def compare_mod_name_coordinates(data, mod_name):
+    """Compare ordering of mod_name against data coordinates, ignoring time.
+
+    First dimension for `data` needs to be the main DateTimeIndex.
```

Comment:
So... you moved it to the bottom.  What I meant was make it an evaluation method in the test class and probably make `data` a class attribute that the tests update after setup.  If it's used in more than one test class, have one test class inherit the other.

Reply:
If you're going to use this in the `extract` module, it belongs outside the test suite and in a new module called `testing` or similar in `utils`, just like `pysat`.

Reply:
`compare_mod_...` was developed for use in utils.extract since the coordinate assumptions across several modules is the same and was used by extract functions before the first move request out of utils.extract. If you prefer utils.testing I can move it there.

Reply:
It's good to have a function do that testing, but yeah, it needs to exist outside of the test suite if it's being used by the main code base.  And it will need its own unit tests.